jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -196,54 +193,74 @@ def _on_spectral_unit_changed(self, *args):
                                self.spectral_unit.selected,
                                sender=self))
 
-    @observe('flux_or_sb_selected', 'flux_unit_selected')
+    @observe('flux_or_sb_selected')
+    def _on_flux_or_sb_selected(self, msg):
```

Comment:
Can this be combined with the `_translate` function?

Reply:
since that observe method is the only thing that calls _translate directly i think that would be fine, but i kept them apart for readability, and in case anything in the future wanted to also call _translate directly. up to you!

Reply:
There is precedent for them being separated and I agree that there could be a use case for calling _translate, so I have no qualms about it. I'll go ahead and approve!